custom_model_runner/datarobot_drum/drum/perf_testing.py

```diff
@@ -319,13 +321,20 @@ def _wait_for_server_to_start(self):
             self._timeout = self._timeout - 1
             if self._timeout == 0:
                 error_message = "Error: server failed to start while running performance testing."
-                (stdout, stderr) = self._server_process.communicate()
```

Comment:
`self._server_process.communicate()` blocks until process is terminated, which will never occur.